tests/Microsoft.Bot.Builder.TestBot/Startup.cs

```diff
@@ -97,6 +98,14 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 app.UseDeveloperExceptionPage();
             }
 
+            // NOTE: Uncomment this to force request buffering to test accessing the request body in buffered scenarios (default is always unbuffered)
```

Comment:
I added this inline middleware to the Test Web App project to facilitate the testing of the two scenarios. I'm leaving it here commented out for the default (non-buffered scenario) and easy uncommenting in the future if the buffered scenario need to be tested again.